### DISCUSSION TITLE
Setting variables doesnt work when there are comments in the script.

### DISCUSSION THREAD

**Discussion by JethroCZ** [dbeaver/dbeaver#16347] [Q&A] [answered]
Hi, I might be just dumb, but few days/weeks ago it worked, and now it does not.
I have sql script like this (this one doesnt work when run with ALT+X):
```
/* Some comment */

@set variable1 = 'some variable'
@set variable2 = 'some variable'

SELECT CONCAT(${variable1}, ${variable2}) from dual;
```
This one works
```
@set variable1 = 'some variable'
@set variable2 = 'some variable'

SELECT CONCAT(${variable1}, ${variable2}) from dual;
```

when I try to run it as whole script (execute SQL script ALT+X), it throws an exception, that I have syntax error near the first row and it writes out whole first row.
I found out that after i delete t comment, it runs smoothly as it used to. Where is the problem, is it bug, should I fill in issue?

I belive, that I didnt change any setting, I am not sure, i might have installed some new version of DBeaver. There were some windows actualizations.
DBeaver version is:  DBeaver	22.0.3.202204170718	org.jkiss.dbeaver.core.product
I can copy whole info, if needed.

Any help would be much apresiated.

**Comment by E1izabeth** [comment, marked as answer]
Hi, @JethroCZ! Thank you for writing. It's a bug and we have already fixed it. Fix will be available in 22.0.4 with the next release. If you encounter the same behavior after upgrading to 22.0.4, please, create an issue.

**Reply by JethroCZ** [reply]
Hi, thanks for replay. I will wait for new version.
